hist/hist/src/TF1.cxx

```diff
@@ -2073,11 +2073,100 @@ Int_t TF1::GetQuantiles(Int_t nprobSum, Double_t *q, const Double_t *probSum)
 
    return nprobSum;
 }
+////////////////////////////////////////////////////////////////////////////////
+///
+/// Compute the cumulative function at fNpx points between fXmin and fXmax
```

Comment:
```suggestion
/// Compute the cumulative function at fNpx points between fXmin and fXmax.
```
End first sentence with `.`, because doxygen will then generate the "brief" doc that's used for previews.